driver-core/src/main/java/com/datastax/driver/core/CloudConfig.java

```diff
@@ -1,70 +0,0 @@
-/*
```

Comment:
You could mention https://github.com/scylladb/java-driver/commit/c2d725f0ddd6fba2412567f40bd92de98adc9124 in description of "Remove Datastax cloud implementation methods" commit - it'd give some additional context about the removal (the documentation of the feature was already removed before).